### ISSUE TITLE
34. 再次降世的准备

### ISSUE THREAD

**Issue by Pathwork-In-China** [Pathwork-In-China/Pathwork-Lectures#36]
34. Preparation for Reincarnation
再次降世的准备
Pathwork Guide Lecture No. 34 | July 25, 1958
心路指导讲座 第33篇 | 1958.7.25

34.1  Greetings in the name of the Lord. I bring you blessings, my dear friends, as I always do for each and every one of you here, as well as for all my friends everywhere.
以主之名问候你们。我像往常一样为你们带来祝福，我亲爱的朋友们，我总是为你们在座的每个人，以及所有遍布各地的朋友们带来祝福。

34.2  As promised, I shall speak tonight about the process of birth. I have received permission to do so from higher authority. My dear friends, let me first tell you that whatever I will say can only explain a very small and limited part of this extremely complicated procedure for which the human understanding cannot possibly be adequate. So please understand that my words can only give you a rough outline. They have to be squeezed into the limitations of human language and understanding.
如之前所承诺过的，今晚我将讨论（人）出生的过程。我得到了来自更高权威的允许去做这件事。我亲爱的朋友们，首先我想说的是不论我说什么，它只能解释（人出生）这个极为复杂过程的一个很小、很有限的部分，（因为）人类的理解力不可能充分完整地理解它。所以请注意，我的话只能给你们一个粗略的轮廓。它们的含义必须被挤压进人类语言和理解力的有限框架之内。

34.3  Before I explain the technical procedure of birth, I will have to talk a little about the spiritual and psychological conditions that govern karmic law, although some of these are familiar in broad outlines to most of you. However, since the technical procedure of birth is directly connected with, and dependent on, karmic and spiritual law, I have to show you the connection, so that you will be able to understand birth as such to some degree.
在我解释（人）出生的技术性过程之前，我必须简单说下掌管因果法则的一些精神和心理条件，虽然说其中一些内容对你们大部分人来说总体上是不陌生的。但是，由于出生的技术过程与因果法则和精神法则直接相关，而且以它们为基础，因此我必须向你们展示其中的联系，这样你们才能在某种程度上理解这个过程。

34.4  Your next life, my friends, is already prepared now, in some way, by each and every one of you. It is not prepared entirely, for as long as you live on earth, the last word is not spoken and the chances of determining your next incarnation and effecting a change from what you have prepared so far are strong. Your chances for that are much stronger here than in your existence in the spirit world. There you will still have a chance to make modifications although the chances are smaller. It is more difficult to do this in the spirit world because of the fact that life–if we do not include the spheres of darkness–is easier there than on the earth plane. For that reason, development is slower in the beyond and to effect a change is definitely more difficult.
你们的下次生命，我的朋友们，现在已经由你们每个人自己在某种程度上准备好了。它还没有全部、完整地确定，因为只要你们还生活在世上，还没说出最后一句话，那么决定你下次降世的情况的机会，以及在你目前已经准备好的计划上进行改动的机会还是很大的。这种机会在你活着的时候比你在精神世界中时要大很多。在精神世界中，你仍然会获得做出调整的机会，但这种机会要少一些。在精神世界中对计划做出改动会更困难一点，因为事实是精神世界（如果我们不包括黑暗位面的话）中的生活要比地球位面上更轻松。出于同样原因，人在“之上”的发展更缓慢，要做出改变也毫无疑问更困难。

34.5  Up to a certain point in the cycle of the development of an individual being, decisions cannot be made by the individual. But from a certain stage of development on, each spirit entity has the right, and often even the duty, to make personal decisions about the circumstances of the following incarnation. Depending on the character of the entity, these decisions are not always good. There are people who tend to be lazy; they lack ambition and are satisfied with a certain amount of comfort. They do not particularly strive for higher forms of consciousness and happiness. Their choice may often be an easier life than would be good for them. They still lack the comprehensive view of the purpose of earth existence. On the other hand, an overambitious and overactive person may often choose more than he is ready for. This, too, will hinder progress and cause a temporary setback. Here the entity’s view does not include its own limitations. Similarly, an overoptimistic character may make just as wrong a judgment as an overpessimistic one.
在一个个体存有发展周期的某个节点以前，它都不具备做出决定的能力。但从某个发展阶段以后，每个魂灵实体都有权利，通常甚至也有义务，去做出关于下次降世的具体条件的个人决定。不同的实体有不同的性格，因此他们的决定不一定都是合理的。有的人偏于懒惰；他们缺乏野心抱负，只要得到一定程度的舒适就会满足。他们不会特别地为了得到更高形式的知觉意识和幸福水平而奋斗。因此他们可能总会选择比最适合成长的情况更轻松些的生活情况。他们还不具备对地球生活的意义的完整理解和视角。另一方面，一个野心太大又过于活跃的人，可能经常会选择超出自己实际能力的计划。这种情况也会妨碍正常进步，还可能会造成暂时的倒退。这样一个人的视角还没有考虑到他自己的缺点。另外，性格过于乐观的人跟性格过于悲观的人做出的判断可能都是错误的。

34.6  As I said before, up to a certain degree of development a spirit entity is not capable of making any decisions, because any extreme in character produces a lack of harmony, which must influence proper judgment. Only when the character becomes more harmonized will the decisions not deviate too strongly from the right middle course. When an entity is not yet developed far enough, decisions are made by higher authority. But even in these cases, the entity is consulted first on what he or she would imagine to be the best life conditions for the next incarnation. This is considered a test, and afterward it is explained to the individual why this or that alternative would be bad or even dangerous and why the decisions made by a trained and highly developed spirit offer a stronger chance for advancement. So, there is a schooling connected with the decision-making and planning for the next life. The entity has to learn something from this alone.
如我之前所说，在到达一定程度的发展水平以前，一个精神实体没有能力做出任何决定，因为任何性格上的极端倾向都会造成缺乏和谐的情况，而这必将对准确的判断力造成不利影响。只有当（它的）性格变得更加和谐时，（它做出的）决定才不会过于偏离正确的中间路线。当一个实体还没有发展的足够远时，相关的决定是由肩负更高职责的实体代为做出的。但就算在这些情况下，当事实体会首先就自己的意见被咨询，看看他或她想象中对于下次降世的最理想情况是怎样的。这会是一项测验，然后会解释给他或她，为什么这种或那种其他选择会是不好的甚至是危险的，以及为什么由经过训练的、高度成熟的魂灵代为做出的决定会提供更好的发展机会。所以，在这种为下次生命做决定和计划的过程中，会包含一个学习教导的过程。相关实体必须独自从其中学到某些东西。

34.7  I may add that even when, on the whole, a being is not yet able to make its own decisions, it may still sometimes have sound ideas and these are then adopted and those wishes granted, while others must be rejected. There is no definite borderline between the state when decisions are made for an entity and when it is ready to make its own decisions. The ability comes slowly and gradually. From incarnation to incarnation, something is learned in this particular respect too, so that more and more ideas of the entity can be considered.
我可以补充的是，哪怕当一个存有从整体上来说还不具有独自做出决定的能力时，它有时仍然可能提出某种合理的想法，这些想法会得到采用，相关的愿望也会得到允许，而其他不成熟的想法必须被拒绝。在一个实体需要别人为它做决定的状态与它能独自做决定的状态之间，没有明确的界限。这种能力是慢慢、逐渐产生的。从一次降世到另一次降世，人们也会在（做决定）这方面学到某些东西，以至于会有越来越多来自这个实体的想法能够得到考虑。

34.8  Then there are also cases when the choices made by the entity are still within the realm of possibility, but may prove too difficult–or too easy. In such instances, the higher authority, advising and planning the next incarnation with the spirit to be incarnated, will advise and explain and offer another alternative. The entity then has the right to accept or to reject the advice according to its own free will. Should the entity reject the advice and then, after the next incarnation, find out that its own judgment was faulty, this then will be a good lesson and maybe the only way it could have been learnt. It would never have been convinced of having erred if not left to follow through on its own ideas. Only when a case is too crass, too hopeless, are the wishes disregarded and the type of lesson to be learned, namely to go through one’s own mistakes, is postponed for later opportunities or, perhaps, incorporated into other details of the next incarnation. So, you see, a great many details and variations are possible even in this one respect.
然后，也有的情况是，当事实体做出的选择仍然处于可能范围之内，但实际上确实太过困难——或太过简单了。这种情况下，那个为将要降世的实体提建议、做计划的更高权威，会建议另一种备选方案并做出解释。这时，当事实体有权利根据它的自由意志选择接受或拒绝这条建议。如果它拒绝这条建议，然后在下次降世结束之后，发现自己的判断力并不准确，那么这次经历既就会成为一个很好的教训，而且这可能也是他唯一得到这种教训的方式。如果没有获得根据它自己的想法去行事的机会，它就永远不会相信自己（在判断上）犯错了。只有当情况过于粗笨、过于无望时，相关愿望才会被无视掉，而这种“把自己的错误进行到底”的教训只好延迟到下次机会进行，或者，也许会包含到这次降世（生活）中的其他细节中去。所以，你们看，就这一个方面里也有非常多的细节和可能的变化种类。

34.9  Needless to say, according to failure or success of the life plan–whether the spirit to be incarnated or the higher authority planned it–development is speeded up or slowed down. Now, since life on earth is one long chain, and each life is nothing but a small link in it, the evaluation of the duties and tasks in each life depend on many, many circumstances. As I mentioned before, each being in existence has a record, called the Book of Life, wherein everything is inscribed. From the time this particular entity has been created up to the present, its special talents, inclinations, characteristics, and the personality trends that brought it to fall are inscribed and updated. Recorded also are all the various stages of development since the fall, not only the earth incarnations, but also all activities in-between. Each incarnation is planned carefully and meticulously according to this “general ledger.”  Before each incarnation, while planning it, the whole ledger is shown to the entity. Whether or not the entity is ready to make his or her own decisions, the purpose of what should be accomplished in the following incarnation is revealed.
毋须多言的是，不管是由即将降世的魂灵还是更高权威做出的，这个生命计划（执行情况）的成功与否将决定这个人的发展进程是被推进还是被拖慢了。同时，由于地球生命（总体上）是一条长链，每次生命不过是其中一环而已，因此评估每次生命中的责任和任务（的进行情况）取许多、许多因素。如我之前提到的，每个存在着的个体都拥有一份记录，叫做“_生命之书_”，其中铭刻着它的完整发展轨迹和所有细节。从这个实体被创造出来开始，到目前这一刻，它的特殊天赋、倾向、性格特征，以及把它引入堕落的人格趋势，所有这些都有记录并且不断更新。同样记录在案的，还有它从堕落开始以来，所有的不同发展阶段，不仅是地球降世生命的这部分，还包括降世之间的所有活动。人每次降世都是根据这份“总帐本”来进行小心细致、一丝不苟的计划的。每次降世开始前，在做计划的过程中，这份完整账本会被显示给即将降世的实体。不管这个实体是否有能力独自做出决定，在这次降世中需要完成的任务和目的都会显示给它。

34.10  The law of cause and effect–or the law of karma–does not necessarily work immediately in the successive life. It often happens that a cause in one life will produce its effect perhaps only three or four incarnations later. For no one is asked to bear too much at a time. I might add, however, that generally, though not always, the higher the development, the faster the effect will follow the cause. Thus, you will perhaps understand a little better when I admonish you not to judge, not to compare, and not to generalize in these matters. You can see only this one life–your own or that of another person–and even this view is a very limited one. That is why a human being is extremely foolish to try to judge the justice of the laws of God.
因果/业报法则——即俗称的karma法则——不一定会立刻在下次生命中产生作用。通常的情况是，一次生命中的一个“因”可能会在三四次降世之后才会产生它的“果”。因为没有人必须要一次承担太多东西。但我要补充的是，一般（但不是所有情况）来说，（个体的）发展层次越高，“因”引起的“果”就会越快显现出来。这样你们也许能更好地理解（为什么）我劝诫你们在这类问题上不要（随意）评价，不要比较，不要（试图去）归纳总结。你们只能看到目前这一次生命——自己的或者另一个人的——而就连这个视角本身也是非常有限的。这就是为什么，一个普通人想要去评判上帝法则的公正与否，会是极为愚蠢的。

34.11  You often feel inclined to say that something is too difficult for you to bear, or that another person has too easy a life. You would not think so for one moment if you knew the missing part of the information. It is for good reasons that, for the sake of your own spiritual development, the curtain must be drawn until you have worked yourself up to a state of consciousness where information of this sort is good for you and for those around you. Therefore, have the humility to withhold judgement. May this be a lesson to all my friends who hear or read these words:  Never judge, never compare your life with that of another, or the fates of two people you know. If you seem to bear a heavier cross, it is because more can be expected of you. You are stronger; you are already farther up the ladder. Or it might also be that you, if you happen to be an ambitious person, have chosen an unnecessarily difficult life, perhaps even against the advice of an angel of God. Consider that, my friends.
你们经常会倾向于说，有些事情对你来说太难承受了，或者另一个人的生活太容易了。如果你们知道，这当中缺失的那部分信息是什么，就一刻也不会这么觉得了。出于很正当的原因，同时也是为了你们自己的精神发展考虑，这种（挡住完整信息的）帘幕必须存在，直到你们通过努力使自己进入更高的意识境界，使这种信息能对你和你周围的人产生正面效果为止。因此，（暂且）以谦逊态度来保留自己的（主观）评价。希望我所有听到或读到这些话的朋友们都能以此为训诫：永远不要（擅自）评价，永远不要拿自己的生活与另一个人的进行比较，或者去比较两个你认识的人的命运。如果你似乎背负着一份更重的十字架，那是因为你背负的期望更高。你更坚强；你在路上前进的更远。也有可能的是，如果你恰巧是一个野心勃勃的人的话，你自己选择了一个不必要地困难的生活，也许甚至是在与上帝的天使给出的建议相反的情况下这么做的。考虑下这一点，我的朋友们。

34.12  I started this lecture by telling you that you are already preparing your next incarnation now. I spoke the truth. How much of the plan you now fulfill determines what opportunities will be afforded you, in your next life, to turn to other things necessary for your overall development. If you do not advance this time at all or do so only partly, the purpose of your present incarnation will have to be repeated at least in part. On the other hand, it also happens occasionally that a person fulfills a little more than he or she set out to do, or brings the plan to fulfillment before the life span is over. In this case, what was meant to be accomplished in the following incarnation can already be started now. It goes without saying that this also changes the plan for the next incarnation, since that has been drawn only roughly as an overall blueprint, subject to change in any direction until the time of incarnation actually arrives. Again, beware of rash judgment; an easy and pleasant life is not always an indication that your last incarnation was a very good one. It may be the result of merits you have earned three or four incarnations ago. Similarly, a very difficult life may or may not be the result of actions in your last incarnation. Furthermore, a demerit may be paid off in one lifetime by a zealous person, while another pays off just as great a demerit, or perhaps even a greater one, in several stages. This, too, makes it impossible for you to judge and compare.
我在这次讲座一开始就告诉你们说，你们现在已经在为自己的下次降世做准备了。我说的是事实。你们目前完成了（本次生命）计划内容的多少，将决定自己下次生命中将得到多少机会，去转向自己整体发展中的其他必要内容。如果这次生命中你们一点都不进步，或者只进步一小部分，那么（下次降世中）你进行本次降世的目的就必须进行至少一部分的重复。另一方面，有时一个人也会或多或少地超出自己本来的计划，或者在生命周期结束前就完成了计划。这种情况下，本应下次降世中完成的任务现在就已经可以开始了。不用说，这自然也会改变下次降世的计划，因为那份计划仅仅是作为一份总体蓝图，粗略地起草好了，在降世实际发生之前的任何时间都可以进行修改。同样地，请注意不要进行轻率的判断；一个轻松惬意的生活并不总能说明你的上一次降世是十分成功的。它可能是你三四个降世之前赢得的美德/正面业报造成的结果。同样，一个非常困难的生活可能是，也可能不是由你上次降世中的行为造成的。另外，一个充满热情的人可能会在一次生命中偿清一份负面业报，而另一个人可能分数个不同阶段偿清一份同样大的负面业报，甚至是一份更大的负面业报。这一点，也使你不可能进行准确的评价和比较。

34.13  All this should be both food for thought and material for meditation. The meditation should make it clear that you have many chances on this earth to make changes for the better in your personal life plan. It is up to you to annul accumulated bad karma. You can do it quicker if you fully realize what this life is all about.
所有这些都应该作为进行思考的食粮和冥想的材料。（对此进行）冥想应该能使你清楚地知道，自己在这个地球位面上拥有许多机会，去在你的个人生命计划中做出积极变化。你可以自主决定，是否采取行动废除之前积累起的不良因果报应。如果你完整地意识到这次生命的目的和背景，你就能更快地实现这一点。

34.14  When life on the earth plane is finished, an account is made; everything is minutely and justly checked so that there can be no difference of opinion or argument about it. For everything is out in the open in the spirit world. It cannot be that one opinion stands against another. Your deeds, your thoughts, your feelings, your reactions and your attitudes create definite forms that are as substantial and visible as any material object is to you at present and even more so. As two normal people would not argue whether a round table in front of them is round or square–as though this were a matter of opinion–the forms perceptible in spirit cannot be argued about. Do not misunderstand me:  there is no prohibition of argument. It is not that the spirits of higher development have a dictatorial attitude, but that you can see for yourself the truth right in front of your eyes, no matter how much you have tried to deceive yourself as long as you were hidden in matter. When the full account is made and carefully considered, it is compared with the original plan for the past incarnation, as well as with the overall plan for the entity in question. The negative trends and faults that were overcome and the tasks that were fulfilled are checked off. What still remains to be done will determine the next incarnation.
当地球位面上的生命结束时，会产生一份记述总结；一切都详细准确、公正恰当地得到核对，以至于不可能出现任何不同意见或看法的分歧。因为精神世界中，一切都处在开放状态。不可能出现两种意见相悖的情况。你的行为，你的思想，你的感觉，你的内心反应和你的情感态度会制造出确定的形态，它们是实体的、可见的，就如目前你们周围的任何实体物质一样，甚至还会更加真实。两个头脑正常的人不会争论他们面前的圆桌到底是圆的还是方的——就好像这是一个个人意见问题一样——在精神世界中，对形状的感知也不是一个能引起争论的问题。不要误解我：精神世界中不存在对争论的禁止。不是说发展程度更高的魂灵会采取一种独裁专制的态度，而是说，不管当你隐藏在物质世界中时多么努力地欺骗自己，（当你处于精神世界中时）你可以自己看到就在眼前的事实。当这份记述总总结成型，并经过仔细考虑后，它会被与上次降世的原本计划以及当事人的整体发展计划进行对照。其中已经被克服的负面趋势、缺点和已经完成了的任务会被清点、勾选掉。剩下未完成的内容将决定下次降世的情况。

34.15  A comparatively long time, according to your human standards, elapses before the next incarnation begins. There is a time of rest for many spirits, particularly for those who have suffered a great deal, physically, mentally, or otherwise. The time of accounting I have just mentioned can take place before the rest period or after. And then begins a period in the spirit world when the entity goes through certain schools, receives instruction according to its personal needs, and where it is put into a particular purification sphere, again according to its individual needs. The sequence of these different phases varies. There is no rule governing which phase comes first and which follows after.
从一次降世结束到下次降世开始前，之间会有一段你们人类的标准衡量下比较长的时间。对许多魂灵来说会有一段休息时间，特别是对那些在身体、心理和其他层面遭受了非常多痛苦折磨的魂灵们更是这样。我刚才提到的进行记述总结的过程，可以在休息时间之前或之后进行。然后会开始下一个阶段，这时当事实体会参加某些学校，并根据它的个性化需求收到指导，然后被放置在一种特定的净化位面里，同样这也是根据它的个性化需求决定的。这些不同阶段的发生顺序不一而同。没有规则在掌管哪个阶段先发生，哪个后发生。

34.16  Purification work in the spirit world is often done in connection with incarnated human beings. This may be new for some of you. Let us suppose that the task of an entity during his last incarnation was to straighten out a relationship with another human being–perhaps to learn to love and to accept this other person–and that the task has not been entirely fulfilled. This task can often be completed, as well as possible, when one of the people is still on earth. This can happen in various ways, but I will not go into the details now, because it would lead us away from the subject. We may discuss this at another time.
精神世界中进行的净化工作通常是与在世的人们联系起来进行的。这对你们有些人来说可能是新的认识。让我们假定说，有个实体上次降世时的任务是要理清与另一个人之间的关系——比如说学会给出爱并且学会（更大程度地）接受这个人——但是这个任务没有全部的完成。通常在两者中仍有一人在世时，这个任务仍然可以尽可能好地得到完成。这可以有许多方式发生，但我现在不会进入具体细节了，因为它会使我们偏离目前的主题。我们可以在别的时间讨论它。

34.17  Purification can also take place in particular spheres that again may vary. There are many, many different levels in the beyond. Periods between incarnations can also include a time of service and the fulfilling of certain tasks in the Plan of Salvation, provided the entity has reached a certain degree of development.
净化工作也可以发生在不同种类的位面上，这些位面种类众多。物质世界之上有许多、许多的不同层次。两次降世之间的阶段也可能会包含一段进行服务和在拯救计划内完成某些特定任务的情况，这种情况适用于已经取得一定发展程度的实体们。

34.18  The different phases–such as taking account of the last incarnation and planning for the next, purification, fulfilling of tasks–may be sharply divided or the phases may interact and be combined. Of course, all this applies only to those spirits who have willingly incorporated themselves in the Divine Order. Many are on their own. They are also brought to an accounting, and planning for the incarnation to follow also takes place for them in the beyond, but all this is done on a lower level, in other spheres, and in a different manner. The same principles hold true, but the way they are applied must vary because the free will of the entity to be incorporated into the Divine Order or not must be considered. This decision influences to a great extent not only the entity’s activities in the beyond, but also its incarnations. As your spiritual consciousness is raised, you will come to discern in which category among your fellow-creatures you belong.  As you can see, how the time between incarnations is spent also determines the life to follow. The length of time between incarnations may vary as well. I want to emphasize here that justice is complete in all cases.
这些不同的阶段——比如归纳总结上次降世和为下次（降世）做计划，进行净化，完成任务等——可能会互相之间区分地很明显，也可能它们之间会产生交互，然后合并起来。当然，所有这些只适用于那些自愿把自己纳入到神圣秩序之中的魂灵们。许多魂灵们是独立行动的。它们也会被带入记述环节，并且也在相应的地方为接下来的降世做准备，但所有这些活动都发生在一个较低的层次，或在其他位面里，并以不同方式进行。相同的原则仍然适用，但具体应用方法必须发生相应的改变，因为必须考虑到这些实体关于“是否愿意出于自由意志，把自己纳入到_神圣秩序_当中”的问题。这个决定在很大程度上不仅会影响到它们在“（物质世界）之上”进行的活动，也会影响它们的降世过程。随着你的精神意识知觉不断提高，你会需要区分你在自己的同伴生灵们中间属于哪个种类。你们可以看到，两次降世之间的时间如何度过，也会决定下次生命的内容。这种时间的长短也会根据不同情况而发生变化。我想在这里强调的是，在所有情况下，相应过程都具有完整的公正性。

34.19  For all who still find themselves in the cycle of incarnation, the time comes for rebirth on earth. At that moment the entity is brought again to the authority in charge of its whole cycle, who has arranged previous incarnations with the entity and done the accounting after the last one. Now its particular personal ledger is again examined along with the temporary plans that had been designed for the next life–subject to change according to the attitudes and activities of the being since the plan was constructed.
对所有仍处于降世循环当中的实体们来说，再次出生在地球上的时候都会到来。这时，相应的实体会再次被带到负责它整个循环的责任人那里，这与之前为它做出降世计划，以及记述总结上次降世的是同一位。在这里，它的个人账簿会再次得到检查确认，同时也会检查那些为下次生命做好的临时计划——这份计划可以在起草之后，根据相应实体的态度和活动随时进行改动。

34.20  At this point the final plan is made for the incarnation to come, and this of course takes considerable time. The proper choice of parents, nationality, religion into which to be born, life circumstances, certain phases in life that are called “fated” are considered and set down. All these choices are made in the most practical way. For instance, parents may be chosen to afford an environment best suited to bring out certain disharmonies in the soul. For that purpose the next life needs certain imperfections in the environment, for if everything were perfect, the shortcomings would not have an opportunity to come to the surface. At the same time, the parents may also be karmically linked to the entity, so that a karmic debt may be paid off.
此时下次降世的最终计划已经做好了，这个过程自然会花费相当长的时间。选择怎样的父母、民族、出生时的宗教环境、生活条件以及被称为“命运”的特定生命阶段，这些都被考虑好并确定下来了。所有这些决定都是以最实际的方式做出的。比如，父母的选择可能会考虑到最能为他提供一种能引发他灵魂中某种不和谐因素的环境的情况。为了达到这个目的，下次生命的生活环境中需要有一些特定的不完美之处，因为如果一切都是完美的，那么这些缺点就没有机会来到表面。与此同时，他的父母可能与他有因果/业报上的联系，从而能因此偿清一份因果业债。

34.21  Difficulties and advantages in the life to follow are considered from the point of view of the overall plan. It is carefully weighed how much should be taken on; how much the entity should concentrate in this life on one character trend, or whether it would perhaps be better to concentrate on a heretofore unsolved psychological problem. For entities of higher development–all this is, of course, relative–tasks for the Plan of Salvation are planned so that they may best combine with the entity’s own purification problems. Some talents are chosen to be brought forth; others are planned to remain hidden in this particular earth life. When all has been considered, the entity is given the chance to make its own choice in certain respects, and where it cannot possibly make its own choice yet, it is nevertheless asked how it would choose if it could. As I said before, the choices of the entity are never denied dogmatically, and the reasons for the denial are always conscientiously explained.
下次生命中带来的困难和优势会得到考虑，这种考虑是从整体计划的视角下进行的。人应该承担起多少重量，这是经过仔细衡量的；同样经过仔细衡量的，还有这个人是否应该在这次生命中集中关注某一种性格趋势，还是更应该集中关注一个以前遇到过，但还未解决的心理问题。对发展程度更高的实体们——当然，这些都是相对的——会安排相应的拯救计划的任务，这些任务会与它们自身在净化过程中遇到的问题以最佳的方式结合起来。有些选择的天赋被选择引发出来的；其他天赋按计划在这次地球生命中是要保持隐藏状态的。当一切事宜都考虑妥当后，这个实体会得到在特定方面做出自己决定的机会，而在那些它还完全不可能独自决定的地方，它也会被询问如果真的可以那会怎么选。如我之前所说，相应实体做出的选择从来不是被教条地拒绝的，如果被拒绝，其中的理由总是会得到如实尽责的解释。

34.22  After an intensive period of study, the final plan is made for the life on earth to come. Special spirits are sent to various places–also on earth–to smooth the way and to make preparations by guidance and inspiration. Once in a while they may report back that conditions are not as hoped for and are therefore unsuitable for the purpose of this incarnation; then other karmic ties are used for this life that would have been concentrated on in a later incarnation if the first circumstances considered had turned out as expected.
在一段集中细致的学习时期之后，为地球上的下次生命做出的最终计划就完成了。会有特殊魂灵们被派到各个地方——也包括地球——来使整个过程更加顺利，并通过给出指引和赋予灵感来做出准备。也有的时候，它们会回来报告说情况并不像原先预计的那样，因此并不适合完成这次降世的目的；这时，会有其他因果纽带被利用起来，以进行这次生命，这些新的纽带是如果原本一切顺利的话为之后的降世准备的。

34.23  When the plan is finally set down and the conditions and preparations on earth checked, the entity is led into a different sphere. It is a great, great sphere, consisting mainly of what you might call, in your earth language, a hospital. Spiritual physicians are working there. Many physicians on the earth plane have been incarnated from such hospital spheres existing on various levels. They learn in these spheres. However, a great number of spirits are also active there who are out of the cycle of incarnations, as well as some who have never been in it. They have more responsible tasks. Others learn under them. The hospital sphere consists of several parts. To mention just one–there is one department to which spirits are brought who have died in an accident or whose life ended in another type of sudden and violent death. In such cases their fluid bodies may be injured and they are then nursed back to health, if I may use this expression, before they begin their activities in the spirit world. This is just one of the many parts of this sphere.
当计划最终定型，地球上的条件和准备工作核对无误后，相应的实体会被带入一个不同的位面。这是一个非常非常巨大的位面，主要由，用你们地球的话说叫做，医院而组成。会有精神医师在这里工作。许多地球位面上的医师，都是从存在于不同层次的类似医院位面降世而来的。他们在这些位面里进行学习。但是，也有很大数量活跃在那里的魂灵们已经处在降世循坏之外，还有些是从来没有参与过其中的。它们拥有更具责任的任务。其他的（魂灵们）在它们的带领下学习。医院位面由数个部分组成。举一个例子——有一个部门，当人们在事故中死亡后，或者更坏的情况，当他们在另一种突然的情况中死于非命时就会被带到这里。在这类情况下，人们的（微妙）液体身体可能会受伤，于是在这里得到疗养并恢复健康，如果我能这样表达的话，然后才开始他们在精神世界的活动。这只是这种位面的其中一个部分。

34.24  In a large section the lives on earth are prepared. The entity ready for incarnation appears there before a certain spirit who has high authority and great responsibility. The entity to be incarnated comes in contact with his guardian spirit or spirits; it may be the same one that has guided him or her through several lives before, or it may be a different one. It will certainly be the one who will lead him through the coming life. The guardian delivers the plan to the spirit in charge in this sphere, who again studies the plan carefully. The spirit will then call some assistants who will work on this entity. And again, everything is discussed minutely, and plans and preparations are made.
在一块很大的区域，进行着地球生活的准备工作。将要进行降世的实体会来到一个特定的具有很高的权威和很大的责任的魂灵那里。在这，他会与自己的守护魂灵或魂灵们建立联系；承担守护魂灵角色的，可能是之前引导过他进行了数次降世的同一位，也可能是另一位。但必然是将要带领他经过即将到来的降世的。这位守护魂灵把降世计划传递给负责这个位面的魂灵，后者会详细了解这份计划。在进行了解的过程中它还会叫来一些助手，专注于帮助这位即将降世的魂灵。同样，这里会对所有事宜进行细致入微的讨论，并做出必要的计划和准备。

34.25  My dear friends, I can sense that some of you think that all this is impossible. It sounds too human, too concrete for you to be believable. Yet, as the saying goes:  truth is often stranger than fiction!  And this is the way it is; although again it is different–different from what is on earth and from what you can imagine. The words I use are closest to what I am attempting to describe. All these spheres and hospitals and physicians and higher placed authorities and assistants and ledgers and plans–all this exists. For you have nothing on your earth plane that is not a poor copy of what exists first in spirit, even if in a somewhat different way.
我亲爱的朋友们，我能感觉到你们中有些人觉得这一切都是不可能的。这些东西听起来太“人类”、太实际了，以至于令人难以置信。然而俗话说得好：真相往往比虚构（的东西）更奇怪！而实际上事情就是这样的；但同时它也是不一样的——不同之处在于它跟地球上的事物，以及你们能想象中的东西又不一样。我选用的词语是最接近我想描述的情况的。所有这些位面、医院、医师、更高权威、助手、账本以及计划——所有这些东西都存在。因为没有什么存在于你们地球位面的东西，不是对存在于精神世界的东西的笨拙的模仿，尽管形式稍有不同。

34.26  Now, when the plan has been carefully studied and the preparations made, the entity to be incarnated waits until conception has taken place. For the conception, astrological factors are taken into consideration, as well as a number of other details of which you either know very little or which you completely ignore. Conception by the future parents cannot happen unless it is the will of God. If a conception today is not to serve a definite purpose for the entity to come into your world, it will be prevented by spiritual means, though it may take place next month. And then, according to the so-called genes of the parents, which of course have been studied beforehand and which must again be in alignment with the entity to be born, certain magnetic fields, rays, fluids get to work–partly automatically as an effect of causes set in motion, and partly administered by spirit helpers. This is extremely difficult to explain to you.
现在，当计划研究妥当、准备工作也就绪后，即将降世的实体会等待受孕过程的发生。对于受孕过程，需要考虑到占星学的因素，以及一系列的其他细节，这些细节你们要么知之甚少，要么完全注意不到。未来的父母的受孕只有在上帝意愿的帮助下才能发生。如果今天发生的受孕对即将进入你们世界的实体不具有明确意义的话，它就会被精神手段阻止掉，但它（仍然）可以在下个月发生。另外，根据父母双方的所谓的基因，这些当然已经被事先研究过，并且必须与即将出生的实体的特性相一致，一些特定的磁场、射线和液体会开始顺利运作，这种效果一部分是由已经启动的过程自动引发的，一部分是由魂灵帮手们施行的。要把这个过程完整地解释给你们会是极为困难的。

34.27  The moment conception has taken place, the entity to be born is made unconscious. It is in such a deep state of unconsciousness that a great part of its knowledge will only come back after life on earth has been completed; another part of knowledge will come back during earth life, but only when the entity leaves the body–in other words, during sleep. Also, consciousness will gradually and slowly awaken in an incarnated human being during the process of growing up. That all this can happen in exactly that way is due to preparations with certain types of fluids in the entity that are being made by the experts in the sphere I have described for you. It requires exact measurement and a certain way of treatment of these fluids and a highly specialized form of processing; there is a refraction and an extension of fluids. There are different types of fluids, each representing a different facet of the personality.
在受孕发生的一瞬间，即将出生的实体会被转入无意识的状态。这种无意识状态程度极深，以至于这个实体的绝大部分知识（都暂时丢失了），只有等到这次在地球上的生命结束后才会被重新获得；另一部分知识会在地球生活的过程中回来，但这也只有当这个实体离开身体时才会发生——也就是说，在睡觉的时候。另外，降世状态中，人的知觉和意识会在成长过程中，逐渐、缓慢地苏醒。所有这些过程能如此分毫不差地发生，是因为降世的实体中某些特定的液体由我之前描述给你们的位面中的专家们做出了特定的准备。这个准备过程需要极为准确的测量手段、对这些液体的特殊处理以及一种非常专业的处理形式；其中会涉及到对液体进行折射和延伸。（这里涉及的）液体的种类有很多，每种液体都代表了一种不同的人格侧面。

34.28  Let us talk first about the physical aspects of the human being to be incarnated. The parental genes are made to correspond to and affect the physical shell. In the meantime, the entity to be born is growing in the body of the mother in such a way that its physical karmic necessities can be fulfilled. There is no coincidence possible. Nothing is left to itself. And when it says in the Bible that God has counted every single hair on your head, it is no exaggeration, my friends. You should understand it in this sense. For every smallest detail has to correspond, because it has a meaning, a much deeper meaning and significance than you can possibly imagine. Symbolism does not work the way humanity thinks it does. It is mostly the other way around. Your bodies are often symbols in one way or another of your spiritual development, psychological trends, and so forth. But beware of rules and generalizations here. There are none.
我们首先讨论下即将出生的人在物质方面涉及的问题。来自父母的基因会受到处理，从而符合并影响（孩子的）物质躯壳。与此同时，即将出生的实体在母亲的身体中生长着，这种生长方式使它的物质（层面的）因果必需品得到满足。这个过程没有任何巧合可言。没有任何东西是随随便便发生的。《圣经》中说上帝把你头上每一根头发都数清楚了，这不是一种夸张的说法，我的朋友们。你们应该从这个意义上理解这句话。因为每个最微小的细节，由于它们本身有各自的含义，因此也都必须对应于另一种远为深层的意义和重要性，这是你们不可能想象到的。象征主义并不是像人类认为的那样方式运作的。实际情况基本上是完全反过来。你们的身体通常是你们精神发展程度、心理趋势和其他因素的某种形式的象征。但要注意避免假定出想当然的规则和一般性规律。这里没有（所谓的）定律和一般情况。

34.29  So the genes are worked on in such a way that the shell is prepared properly. Some genes will therefore have an influence; some will not. In one instance the genes of the mother have a greater influence, in another those of the father. In one instance genes of the parents do not work at all, in another case, the genes of a grandparent, or even a great aunt may all of a sudden be at work. All this is not arbitrary or left to coincidence. For every detail there is a reason. The physical shell grows in the mother’s body, although there is no spirit in it, but the procedure or preparation proceeds according to the rays, the magnetic fields that work their effect corresponding to the purpose of the treatment in this particular sphere; everything takes place exactly according to plan.
于是（来自父母的）基因得到了相应处理，（孩子的）躯壳也因此准备妥当了。有些基因会产生它们的效果；有些不会。一种情况下来自母亲的基因会有比较大的影响，另一个情况下来自父亲的基因影响力更大。有时来自父母的基因完全不会显示出来，也有的情况下来自祖父母的基因，甚至是来自某个姑妈的基因可能会突然产生作用。所有这些都不是随意而为或者是碰巧发生的。每个细节背后都有相应的原因。（孩子的）物质躯壳在母亲的身体中生长着，尽管现在还没有魂灵存在于其中，但相关的步骤或准备工作都在根据相应的射线和磁场进行着，这些射线和磁场会根据这个特定位面中的处理方式的目的，产生所需的效果；一切都分毫不差地按照计划进行。

34.30  Also, many other factors are taken into consideration. Certain highly trained specialists concentrate on the physical shell. Other specialists, equally trained, concentrate on psychological and spiritual factors. It occurs also that through the treatment of certain other fluids some consciousness will return as the human being grows while other types of consciousness must remain hidden. Thus, one person will grow up with a strong feeling of God-consciousness, perhaps even in spite of atheistic and materialistic parental and other influences. In another case, it is the other way around. In some instances, there is a strong urge to become, let us say, an artist or a scientist, in spite of contrary environmental influences in youth. This is due to the effect of certain fluidal matter that has been prepared before birth. This preparation determines what urges, trends, and leanings should come to the surface at what period. Other fluids are prepared and processed in a way that is conditional on whether other factors have been fulfilled or not. You can perhaps vaguely glimpse how infinitely complicated and exact this preparation of the so-called soul must be.
同时，许多其他因素也都考虑在内。一些受过非常专门训练的专家会专注于物质躯壳方面。另一些受过同样程度训练的专家会专注于心理和精神因素。由于某些其他种类的液体得到了处理，特定种类的知觉意识会在人成长的过程中再次浮现，而其他类型的知觉意识必须保持隐藏的状态。因此，有时候一个人会在成长过程中带有一种很强的上帝—意识的感觉，就算受到无神论、物质享乐主义的家庭环境的影响和其他影响也会如此。也有时候，情况刚好相反。某些情况下，有的人内心中会有一种成为，比如说，艺术家或科学家的强烈推动力，尽管他在年轻时受到过相反的环境影响。这都是出生前准备好的某些特定液体物质的效果。这个准备步骤决定了人在哪些时期会有哪些冲动、趋势和倾向出现在（意识）表面。也有另外种类的液体经过其他方式的准备和处理，会根据相关因素的完成与否，条件性地做出反应。（从这里）你们也许能模糊地瞥见，这种对所谓的“灵魂”的准备工作必须是多么地无限复杂和精确。

34.31  As you know, the faults that have been planned for the person to work on in this life are energies or currents. These are treated in the fluidal bodies in such a way that they need very little outer challenge to come to the surface. The outer challenges are provided for in the choice of parents and other conditions in the incarnation. If faults remain dormant in the depth of your soul, you cannot become aware of them, and if you are not aware of them, you cannot overcome them. Not all problems or faults are prepared for raising to the surface. Some are supposed to remain hidden, to be worked on in a future incarnation or, as I said, only if and when the plan of this life has been fulfilled. All this must be carefully studied in the plan by the experts and prepared properly.
如你们所知，按照计划，人在这次生命中需要处理的缺点（本质上）是一些能量或者说涌流趋势。这些能量和涌流趋势在人的流态身体中经过了特定的处理，以至于它们不需要太多的外部挑战因素就能浮到表面。外部挑战因素是通过对父母及一生中其他条件的选择而产生作用的。如果你的缺点在自己的灵魂深处保持潜伏状态，你就无法觉察到它们，而如果你觉察不到它们，你就无法战胜它们。不是所有问题和缺点都被准备好浮到表面。有些应该保持隐藏状态，等到未来的某次降世中再进行处理，或者，如我之前说的，只有当这次生命的计划目标实现后才可能出现。所有这些都必须由专家们在计划中进行仔细研究，并做出妥当准备才行。

34.32  The same applies, of course, to talents, to psychological problems and other complexities. Some are prepared so lightly that they will arise immediately, others only after definite conditions are fulfilled. This preparation in the fluidal bodies will perhaps be easier to understand if you think of each quality, each trend, each attitude, good or bad, as a power current, which it truly is. It depends very much on how the power currents are treated, insulated, or channeled.
当然，相同的情况也适用于人的天赋、心理问题和其他复杂问题。有些东西只进行了很简单的处理，这样它们立刻就会显现出来，其他问题只有在达到某些明确条件后才会出现。如果你们想更好理解这种对流态身体的准备工作的话，也许你们可以把人内心每种品质、每种趋势、每种态度，不管好的还是坏的，想象成一种能量涌流，因为实际上它们确实是。事情很大程度上取决于这些能量涌流得到了怎样的处理、隔离或引导。

34.33  All this may serve as an explanation for some of you who have seen in your family or in your environment a great change in someone who has always been known to be a very decent person, conscious of his shortcomings, working at them, and obviously making great spiritual progress. All of a sudden something unexpected and apparently quite new comes to the surface. Everyone is shocked and disappointed. From now on you will understand that this is often due to the fact that this person has worked spiritually very well. After work has been accomplished on trends that were on the surface, other trends, left deeper embedded by the spirit helpers, can now emerge to be worked on in the purification process. This is exactly why people on a spiritual path often display at certain periods trends that have never been noticed before. Of course, this may also be due to their having been hidden by the mask self. On the other hand, these trends may have been left to emerge only after certain other things have been accomplished. The same may apply to a good trend or a talent. The spirit specialists examine their charts exactly. In one case a strong talent is supposed to come out very soon and it has to be prepared. In another case, a talent is left inactive, and may only emerge depending on certain circumstances and on whether or not the person is following through on the plan. All this is shown on the chart, and the fluids have to be prepared and processed accordingly, which is an exceedingly difficult procedure impossible for you to grasp entirely.
这里提到的内容你们中有些人来说可以作为一种解释。你们在自己家庭或周围环境中都曾看到过：有一个人们熟知的、从来都很体面又正派的人，他很清楚自己的缺点，并努力地克服它们，他明显在精神修养方面取得了很不错的进展。突然之间，某种人们意料之外、并且显然很新鲜的东西浮出水面。所有人都对此感到震惊和失望。现在你们会知道，实际上这说明这个人在精神层面取得了可观的进展。人们性格表面的趋势涌流得到妥善处理后，就会有其他的、被精神世界的帮手们埋藏更深的趋势涌流（适时地）浮出水面，成为净化过程中新的工作材料。这也正是为什么，人们在某些特定时期会表现出某些他们之前从没注意到过的趋势或行为。当然，这种现象也可能是因为人们的面具自我之前一直把这些趋势涌流隐藏了起来。另一方面来说，这些趋势涌流可能本来就是要等到某些条件达到后才会浮现出来。这个道理也适用于那些正面的趋势涌流，或者某种天赋。精神世界的专家们会准确无误地检查它们的图表。有时候，一项很强的天赋应该在生命中很早的时候就展现出来，因此它必须得到相应的准备。也有时候，某种天赋被保留在未激活状态中，而且只会根据特定的条件以及这个人是否在根据计划行事，来决定是否浮现出来。所有这些都显示在图表上，而相关的流体必须得到相应的准备和处理，这是一个非常困难的步骤，你们想要完全弄清楚这个步骤是不可能的。

34.34  The period of processing and treatment of the fluidal bodies takes approximately nine months, the time the human shell needs to grow. Sometimes the processing is completed a little before the nine months are over; then the entity is just waiting in an unconscious state. Sometimes life needs to start a little earlier because the conditions to begin life may be more suitable, and the parents also fulfill the best conditions for a premature birth. In such a case a premature birth will occur. In other instances, birth may be delayed a little.
对流体态的各个身体进行加工和处理的时间大概是九个月，这也是人类躯壳生长所需的时间。有时候这种处理比九个月的时间稍微短一点；这时相应的实体就会处于一个无知觉的状态中，静静等待。有时候生命需要稍微提前一点开始，因为此时开启生命的相关条件更加适宜，而且父母也满足进行早产的最佳条件。这种情况下早产就会发生。也有的时候，出生会略微耽搁一段时间再发生。

34.35  At this point I would like to add one more detail. During the time of pregnancy, the attitude of the mother, as well as that of the father, may often determine a change. If the parents, or even one of the parents, should change their spiritual attitude, then the entity first planned for these parents would not fit into those circumstances any more. This may be because, if their spiritual attitude has changed for the better, the parents now deserve a higher-developed spirit as their child. It may also be that the being first destined for these parents will not find the necessary fulfillment with parents having a spiritualized outlook. Enough parents are available whose attitude would offer the limitations that the particular entity needs for growth. Fewer, indeed, will present the proper environment for a spirit who has more to give to the world. This is also a task and a responsibility for the parents. They have to be worthy to rear a child who has greater tasks to fulfill. Such worthiness is often determined by the spiritualized way of looking at life.
在这里我想再添加一个细节。在怀孕的时间里，母亲持有的态度，以及父亲的态度，通常会决定是否会有变动发生。如果父母双方或其中一方改变了他们的精神态度，那么本来准备好要成为他们孩子的实体就不再符合相关的情况了。这是因为，如果他们的精神态度转向了更积极的一方，那么这对父母现在就应该得到一个发展程度更高的魂灵作为他们的孩子。也有可能的是，一开始注定成为这对父母孩子的实体在具有精神层面视角的父母这里，将得不到它所需要的环境和挑战。（地球位面上）有足够多的父母候选人，能提供这个特定实体成长所需要的必要局限条件。但确实，能为一个可以给这个世界带来更多价值的魂灵提供合适环境的父母人选们就要少些了。这对父母来说既是一项任务也是一份责任。他们必须符合一定条件才，可以抚养一个有更重要任务需要完成的孩子。是否符合相关条件通常取决于他们是否有以精神层面的眼光看待生活的处世方式。

34.36  So, last-minute changes can be made if the mother, or the father, or both, change their attitude during pregnancy, either for the better or for the worse. Enough possibilities are available for such changes to be effected. The spirit world can foresee fairly well whether or not such changes are within the realm of the possible, knowing the capacities of each soul much better than the human beings themselves, because the Book of Life is available there. So tentative arrangements may be made already during the time of pregnancy. For instance, should the first chosen parents change enough during the time of preparation for birth to warrant their receiving a different child, the first intended child will go without difficulty or delay to another pregnant woman. Therefore, it is advisable for the mother in particular to become quiet within, to turn to God, and to spiritualize her whole being and attitude during pregnancy.
所以，如果母亲，或父亲，或父母双方在怀孕期间改变了他们的态度，不管是朝好的方向还是朝坏的方向，这都有可能导致最后一刻发生必要的调整和变动。有足够多的可能性能够导致这样的变动。精神世界可以相当清楚地提前看到，这样的变动是否存在于可行的范围之内，同时对每个人灵魂潜能的了解也比人们对自己的了解准确很多，因为精神世界中可以查阅到_生命之书_。所以在怀孕的时间里就已经有试探性的安排做出了。比如说，如果首选父母在孩子准备出生的时间里发生了足够多的改变，以至于他们理应获得另一个孩子，那么那个本来要给他们的孩子会很快被顺利地转交给另一位怀孕的母亲。因此，对这位母亲来说明智的做法是，在怀孕期间使自己内心变得平静，转向上帝，使自己整个身心都变得更加精神化。

34.37  When finally the time of birth arrives, many spirits assist, so that the fluidal bodies that have been treated in the manner described enter the body of the baby. Then the incarnation has been effected. Are there any questions about this?
最后，当出生的时刻来临时，许多魂灵都会提供协助，使那些由之前描述的方法处理过的液态身体进入婴儿的身体中。于是降世就实现了。对这个过程有什么问题吗？

34.38  QUESTION:  Yes. How is it possible that sometimes babies die at birth after they have already received their souls, so to say?
问题：是的。为什么有时候婴儿出生时会，可以说，在已经接受了它们的灵魂后死去？

34.39  ANSWER:  There are many possible reasons for that. In the first place, it may be that it is a karma for the parents, a test through which they have to pass. But the spirit of the baby will not have gone through this procedure for nothing. That the first mother would lose her child was known in the spirit world, and the parents with whom the baby would finally settle down have already been chosen before the first birth occurred. The spirit of the baby will thus have fulfilled an additional little task by going through this experience, even though it is unaware of it since it is in a state of unconsciousness. Nevertheless, this is not done without the consent of the spirit to be incarnated before it undergoes the birth process. It will mean a merit for the spirit who makes itself available as an instrument of fate, so to speak, for the parents it so shortly visits.
回答：这有许多可能的原因。首先，这可能是父母的因果业报，是他们必须经过的一个测验。但婴儿的魂灵不会徒然地经历这一切。精神世界知道，第一位母亲会失去她的孩子，而最终将安顿下这个婴儿的父母，是在婴儿第一次出生前就选定了的。这个婴儿的魂灵通过经历这个过程也完成了一项额外的小任务，而由于它处于一种无意识的状态中，它对这个过程并无觉察。但无论如何，这个过程不是在没有获得将要降世的魂灵的允许下进行的。这对它来说会是一份功德，因为它使自己成为了自己如此短暂地拜访的父母的，如果可以这样说的话，命运的工具。

34.40  There may be other reasons too. The possibilities are numerous. For instance, the parents may not have lived up to certain promises made in the spirit world. However, it may also be that the parents have fulfilled more, spiritually speaking, than was expected of them and thus are ready to bear this cross and test which otherwise would have been too much for them. This will further their development. So such a thing may either happen because the parents fulfill less or because they fulfill more than originally planned. In either case, later on something better may follow. As far as the baby is concerned, after its physical death there will be a comparatively uncomplicated process whereby the fluids will be retracted a little, certain adjustments will be made, and without gaining consciousness, birth will occur with the other parents, chosen as permanent hosts in the first place.
这其中也可能有其他原因。相关的可能性有许多。比如，这对父母可能没能达到他们之前在精神世界中做出的承诺。但是，也有可能这对父母在精神层面超额完成了他们分内的任务，因此现在准备好了承担起这份本来超出他们能力的负担和测验。这会促进他们的精神发展。所以这样一件事，既有可能因为父母比原计划完成的更少而发生，也有可能因为他们完成了更多而发生。不论哪种情况，后面可能会有某种好一点的事情随之而来。对婴儿而言，在它物质身体死亡之后，会有一个相对来说不太复杂的步骤，这时会有一部分液体被撤回，然后还会进行一定的调整，之后在它仍然处于无意识状态的情况下，出生过程会在一开始就被选定为最终家长的父母那里发生。

34.41  QUESTION:  We have now two and-a-half billion people on this earth. According to any reliable census, this is the maximum we can have. This must mean a rapidly increasing number on the other side. Now, probably every human being has the same allotment, or is that not so?
问题：现在地球上有二十五亿人口。根据任何可靠人口普查的说法，这个数字是我们能支持的最大值。这必然意味着“另一面”的人口会快速增加。那么，应该每个人都有相同大小的人口配额，还是说并不是这样？

34.42  ANSWER:  Certainly. You see, my dear friend, the Spirit World is so vast that you cannot have an idea of it. There are so many worlds, so many spheres, and so very very many spirit beings, of all types of development. There is really enough for everyone. You can be quite sure of that. There are many many more spirits than the number of human beings that will ever exist.
回答：当然。你看，我亲爱的朋友，_精神世界_是如此广大，以至于你无法对它产生恰当的概念。宇宙造物中有如此多的世界，如此多的位面，以及许多许多许多的精神存有，它们的发展类型各不相同。每个人真的都有足够的东西。你可以相信这一点。精神魂灵的数量比所有存在过和将要存在的人类数量要多得多。

34.43  QUESTION:  Is it true what Theosophy teaches, that the reservoir of spirits to be incarnated holds about three times the number of the present earth population? spirits to be incarnated?
问题：神智学教导说，将要降世的魂灵的存储量大概是目前地球人口数量的三倍，这是真的吗？将要降世的魂灵数量？

34.44  ANSWER:  My dear friend, I have not counted them and I am not curious to know the number. So I could not possibly give you an answer here. I do not know it. If it would be important, I would make my inquiries.
回答：我亲爱的朋友，我没有计数过它们，我对这个数字也不很好奇。所以我不可能在这里给你一个回答。我不知道这个数字。如果这是重要的事情，我会提出我的询问。

34.45  QUESTION:  With regard to the “Pistis Sophia,” we have read that the so-called servitors are working on that. Now that would correspond to these many entities…
问题：关于《救世主的智慧》这本书，我们从中读到所谓的服务者正在处理这件事。这就对应了如此多的实体们…

34.46  ANSWER:  Exactly. So I think that takes care of that question.
回答：正是。所以我觉得这就解决了你的问题。

34.47  QUESTION:  Is it correct that birth, and time and form of death, are determined by an unchangeable destiny?
问题：一种说法是，人的出生以及死亡的时间和形式是由一种无法改变的命运决定的。这种说法正确吗？

34.48  ANSWER:  No, it is not correct. As to birth, I have just explained how it is, so I do not have to go into that any further. And, as you have heard, even that can be changed in the last moment. And the time and circumstances of death can also be changed. As I said to you, there is a plan for each life. But I also told you that there are many plans. There is a plan for every alternative, every possibility of free-will decision.
回答：不，这种说法不正确。关于（人的）出生，我刚刚已经解释了它是如何运行的，所以我不需要在继续说下去了。而且，你也听到了，就连那个也可以在最后一刻发生改变。而死亡的时间和情况也可以被改变。如我之前对你说的，每次生命都有它的计划。但我也告诉你说同时存在多个计划。对每种不同可选项，每种可能的自由意志决定，都有其对应的计划。

34.49  Let us say, an entity fulfills the most he or she was expected to. For that alternative there is one plan. Another plan has been made for the eventuality that he fulfills even more than the best that could be expected. That exists occasionally too. There are several other plans for partial fulfillments, or if he does not fulfill anything. Perhaps a person will straighten out a karmic debt, but will not overcome a certain weakness or solve a psychological problem. The time of death does not vary according to which plan is fulfilled, but the time and manner of death varies according to which plan has come true. Again, I wish to emphasize that the fact that life can be longer than projected in one of the plans is not always a sign that this longer life has been fulfilled at its best, though it may be so.  If an entity really fulfills his or her very best, his life may be prolonged because through his special fulfillment he may have possibilities to help others. But then, again, a life may be cut shorter because the entity has fulfilled his or her best. Then the favor is granted to return home sooner, to begin something new, a task that he or she has been waiting for. Therefore one cannot bluntly say that an unforeseen prolongation of life is necessarily a proof of good fulfillment. It may be that way, but it may also be exactly the opposite. So the time, as well as the form of death, can vary according to which of the many alternatives the entity has chosen.
比如说，一个实体最大限度地完成了自己被期望做的事。这种可能背后，有一种对应的计划。如果他完成了比最大期望值还多的成就，这种情况背后，同样有一份对应的计划。这偶尔也会发生。对部分完成，或者一点都没有完成的情况，相应地有许多种其他计划。也许一个人能够理清一份因果业报，但没能战胜某个特定的弱点或解决某种心理问题。不是说死亡时间是根据哪个计划得到完成而发生变化，而是说，死亡的时间和方式是由哪份计划实际发生而决定的。不过，我想强调的是，“某一份计划中的实际生命长度可能比原来预计的要长”这个情况，不总能说明这份长一些的生命的完成情况是最好的，虽然说这是可能的。如果一个实体真的以最高水平实现了自己（的价值），那么他的生命可能会延长一些，因为通过这种特殊的成就，他也许有机会去帮助其他人。不过另一方面，生命也可能因为这个实体已经以最高水平实现自己（的价值）而缩短。这时它就得到特殊照顾可以提前回家，开始某种新的东西，比如一项他或她等待已久的任务。因此不能直截了当地说，意料之外的生命的延长就一定证明他很好地实现了自我价值。事实可能确实如此，但也可能刚好相反。所以死亡的时间以及形式，可以根据这个实体选择哪种备选可能性而发生变化。

34.50  QUESTION:  This would explain the astrological riddle that sometimes death does not occur according to the usual progressions.
问题：这可以解释那个占星学谜语：有时候死亡不会按照通常的进程发生。

34.51  ANSWER:  Exactly. You see, my dear friend, perhaps now, with the help of the teachings you receive here, you will be able to understand many things in astrology that were not quite clear before. It may be very good to adopt right now the idea that each human being has necessarily been given many alternative plans for a life. This will also clear up another question, namely, why some aspects “work,” as you say, and some do not. That is so because astrology does not contain all plans. Some are found in domains not yet known to humankind. Perhaps you can fit this into your picture.
回答：正是。你看，我亲爱的朋友，也许现在借助你从这里得到的教导的帮助，你能理解占星术中很多之前不太清楚的东西。也许现在你可以接受一个概念：每一个人都得到了针对这次生命的许多备选计划。这也会澄清另一个问题，也就是说，为什么有些方面像你说的那样“起作用”，而有的不起作用。之所以这样，是因为占星学并不涵盖所有的计划。有些计划存在于人类尚未知晓的领域。也许你可以把这一点融合到自己的知识构架中。

34.52  QUESTION:  Last time the question was asked whether a total destruction of entities, souls, took place before the salvation through Jesus Christ.
问题：上次有一个问题问到，在通过耶稣基督的拯救发生之前，是否曾经发生过实体、灵魂的彻底毁灭的情况。

34.53  ANSWER:  There never was a total destruction of a single soul. Never. No created being has ever been totally destroyed, not even Lucifer.
回答：从来没有任何一个灵魂被彻底毁灭过。从没有。没有任何被创造出的存有被完全毁灭过，甚至路西法也没有。

34.54  QUESTION:  Did Dante’s prophecy of the Veltro refer to the coming of the Holy Ghost as predicted in de Fiore’s Eternal Gospel, and is that Veltro taken from the Vangelo Eterno, in other words, the coming of the Holy Ghost?
问题：但丁对灰猎犬的预言是否指的是de Fiore的《永恒的福音》中预言的_圣灵_的到来？同样的那个Veltro（灰猎犬）是否取自Vangelo Eterno（意大利语，永恒的福音），即，_圣灵_的到来？

34.55  ANSWER:  I may answer the question only in the following way–for there are certain spiritual laws that bind me to the kind of statements I am allowed to make. The answer may already be partly clear by my having mentioned these various plans. Different clairvoyants, seers, or otherwise psychically gifted people have seen one plan, others have seen another plan, and that should explain why one clairvoyant can be right in many instances and wrong in others. In the latter case he or she just happened to see a plan that did not come true, due perhaps to an unexpected change on the part of one or several human beings.
回答：我只能以如下方式回答这个问题—因为有特定的精神法则，约束我只被允许做出特定种类的回答陈述。我前面提到了这些不同的计划和可能性，这可能已经部分地回答了这个问题。不同的透视者、先知或其他拥有通灵天赋的人们看到了一个计划，其他人看到了另一个计划，这应该能解释，为什么某个透视者可以在许多情况下是正确的，有的时候却是错误的。在后者的情况下，他或她碰巧看到了一个没能成真的计划，这也许是由一个或数个人那里发生了不可预期的变化导致的。

34.56  Here I would like to say this:  It is, of course, the general plan of development for the whole Plan of Salvation that humanity will come to a point–how soon is difficult to say–when an intermediary human authority between man and God will no longer be necessary. Then the various churches and religious organizations will teach their followers in a way similar to how I am teaching you now, that is, to cultivate and obtain direct contact with the Divine World. This may not necessarily be through mediumship, which is only direct for the medium, but through spiritual development and the refinement of the inner senses. The religions will subtly, gradually, but surely change in the centuries to come. You can observe this trend already. Some time ago, people killed each other just because they thought differently. Yesterday, they hated each other but did no longer kill because of their different ways to come to God. Yet,  whoever was in power would still enforce his belief and prohibit the others. Today, humanity has come far enough, so that people begin to tolerate one another. They still feel that there is something to “tolerate”–the other fellow’s different conceptions–nevertheless, this is a great step forward. Tomorrow they will not feel they have to tolerate one another, rather, they will see the one truth that underlies all religions. Religion will no longer be a separating factor. And that is the Age of the Holy Spirit that is referred to in your question.  This era, in all probability, will not come about by destroying the existing religions suddenly. As Jesus has said, he did not come to destroy. People will change from within, gradually, in the only way positive and constructive growth can ever take place. And this is what was foreseen and expressed in this prophecy. Does that answer your question?
在这里，我想这样说：整个_拯救计划_的总体发展计划是：人类整体会到达一个节点—这能多快发生还不好说—在这个节点后，人与上帝之间就不再需要一个权威的人类中介角色存在了。这时候，各个教会和宗教组织就会以一种类似于我现在教导你们的方法教导它们的追随者了，也就是说，去培养和建立与_神圣世界_的直接联络。要做到这一点不一定是要通过媒介传导的方式，因为这时只有作为媒介的人是在进行直接沟通的，而是要通过（人们普遍性的）精神发展，以及对内心感觉的不断精炼和纯化来达到这一点。不同的宗教会微妙地、逐渐地，同时必然地在接下来的几个世纪中发生变化。你们现在已经可以观察到这种趋势了。不久之前，人们仅仅因为相互之间思考方式不同就会杀掉对方。昨天，他们仍然厌恶对方，但不会因为各自接近上帝的道路不同而下杀手了。但是，不论是谁在掌权，他仍然会强制执行自己的信条同时遏制别人的。今天，人类已经走了很远的路，以至于人们开始容忍彼此了。他们仍然觉得有需要“容忍”的东西—另一个人的不同见解—但不论如何，这都是向前的一大步。明天，他们就不会觉得还需要容忍对方了，相反，他们会看到构成一切宗教信仰基础的同一真理。宗教将不再是一个起到分离作用的因素。而这就是你的问题中提到的_圣灵的时代_。最可能的情况是，这个时代的到来将不会通过突然毁掉现存的各种宗教的方式发生。如耶稣所说，他不是来进行毁灭的。人们会逐渐地从内心发生改变，这也是积极的、有建设性的成长唯一可能发生的方式。而这就是你提到的预言预见到并表达出的内容。这是否回答了你的问题？

34.57  QUESTIONER:  Yes, except the last part, whether the Veltro actually refers to the prophecy by Joachim de Fiore.
提问者：是的，但没有回答最后一段，也就是说灰猎犬到底是不是出自Joachim de Fiore的预言。

34.58  ANSWER:  That I am not allowed to answer. If you study it carefully, you may be able to sense the truth. It is possible that you are inspired. But it would be against spiritual law if I gave you a simple yes or no. Perhaps it is difficult for you to understand why, since this is all new to you. But that is the way it is. You see, it is our purpose to further spiritual development, directly or indirectly, and we are only allowed to answer questions that do that. But if knowing the answer to your question has for your work a particular importance, I would advise you to ask your guardian spirit to inspire you. And you will receive the answer.
回答：那是我不被允许回答的。如果你仔细研究这个问题，你也许能够感觉到其中的真相。你有可能受到灵感启迪。但如果我给你一个简单的是或否，这是违反精神法则的。也许对你来说很难理解为什么会这样，因为这一切对你来说都是新奇的东西。但事情就是如此运行的。你看，我们的目的在于以直接或间接手段推进精神发展，因此我们只能回答能够达到那个目的的问题。但如果知道这个问题的答案对你的工作具有特殊的重要性的话，我建议你请求自己的守护魂灵为你提供灵感。然后你就会收到答案。

34.59  It is God’s will that certain knowledge can only be given to a human being if he makes an effort to attain a personal contact with the Spirit World of God, through his own channels. Spirits of the World of God who talk through a medium can only show you the way how to obtain such personal communication. And this I am doing. (Thank you.)
某些知识只有当一个人做出努力，通过他自己的渠道与_上帝的精神世界_取得个性化的联络时才会被传递给他，这是上帝的意愿。通过一个媒介进行交流的来自_上帝的世界_的魂灵们，只能向你展示如何获取这种个性化、个人层面的沟通。而这正是我在做的。（谢谢你。）

34.60  QUESTION:  My question concerns the prolongation of life through science, the extension of the life span and the elimination of child death. Since everything is so beautifully planned, how is it that science claims to have some part in the extension of life?
问题：我的问题是关于通过科学手段延长寿命，延长生命周期和消除婴幼儿死亡的。既然一切都如此美妙地按计划进行，那为什么科学界声称自己对延长生命做出了一部分贡献？

34.61  ANSWER:  Science can find only what God’s Spirit World allows it to find. This permission depends on two factors:  One is that the findings have to fit into the general plan, concerning the entire earth sphere at any particular given time. The other factor is that the human beings in question, in this case the scientists, make the necessary effort. Effort is always an integral part and constitutes the first step. The same holds true, as mentioned before, for reaching for contact with the Divine World by an individual or a group. If the effort is not made on the human side, if the human attitude does not fulfill the basic requirements, the Spirit World cannot do its part. Similarly, I can speak through this human person only because of her efforts. If she refused to sit down here, I could not do anything to come through here. It is the same with a scientist, or an artist for that matter. With all his talent, he cannot accomplish anything quite by himself. He or she must have spiritual helpers who can manifest only if he is fulfilling the necessary requirements. Now, if science is discovering today so many new things, it is because this fits into the present stage of development. There must always be some leeway to see what humanity does with such findings, whether they are used for good or for evil, selfish, destructive purposes. In the latter case, material progress will ultimately wither away, as it has happened before. So, if life is longer, it is because it fits into the whole picture of overall development. Since humanity as a whole is farther advanced, people may accomplish more in one lifetime–and that works hand in hand with the technical and scientific improvements which, in turn, will only bear fruit if spiritual improvement follows, at least to some degree. So science on the whole, or the scientist individually, is to be taken but as an instrument.
回答：科学只能发现_上帝的精神世界_允许它发现的东西。这种许可取决于两个因素：其一是，相关发现必须能纳入（人类）总体发展计划当中，这涉及到任意给定时间点上整个地球位面的综合情况。其二是，当事的人类个体，在这个例子里就是科学家们，必须做出相应的努力。努力永远是一个核心部分，并且也是最初的步骤。这对于试图与神圣世界建立联络的个人或组织来说同样适用，我们前面提到过这一点。如果人类方面没有做出必要的努力，如果人们的态度不符合基本要求，那么_精神世界_就不能完成自己的工作。同样地，我能通过这个人类个体说话，也是因为她做出了努力。如果她拒绝坐在这，我就没有任何办法来到这里。这个道理对一个科学家，或者艺术家来说是一样的。以他全部的天赋，他没法完全靠自己去成就任何事。他或她必须拥有精神帮手，而这些帮手只有在他满足必要的条件时才能展现效果。所以，如果今天的科学界正在发现如此多的新鲜事物，这是因为这符合（人类）目前的发展阶段（和水平）。（精神世界）总会留有一些余地，来看看人类会用这种发现来做些什么，是用于正面目的还是用于邪恶、自私、毁灭性的目的。如果情况是后者，那么物质进步最终会衰亡枯萎，就像之前发生过的那样。所以，如果（人的）寿命变长了，这是因为它符合（人类）总体发展的全局。由于人类作为一个整体取得了一定的进步，人们可以在一次生命中取得更多（精神方面）的成就了——而这与科学技术的发展是协同的，而科技的进步，反过来，只有在（人们的）精神发展跟得上的情况下，才会结出果实，或至少从一定程度上是这样。所以不管是科学界整体，还是科学家本人个体，都只应当被作为一种工具手段来看待。

34.62  And now, my friends, I will retire. I will not speak to you for a short while but this should not mean, must not mean, that you are cut off from the World of God. You yourself should make an effort to remain in touch with it in your thoughts, your efforts, and in your times of inner quietness. Continue to search in your heart for the spiritual truth you need to solve your problems and your life. The more you seek, the more will be given onto you. You are never alone if you reach out for God’s spirits who are constantly around you, waiting for you. Blessings for all my friends, for your bodies, for your souls, and for your spirits. Be in peace, God bless you, be in God!
现在，我的朋友们，我要退回了。我在一段较短时间内不会对你们说话了，但这并不意味着，也绝不意味着你们被与_上帝的世界_分割开了。你们自己应该做出努力，使自己在内心思想中、日常努力上和内心平静的时间里，与它保持联系。继续在自己内心搜寻你需要的精神真理，来解决你的问题和应对生活（的挑战）。你探求的越多，给予你的就越多。如果你伸出手去寻找那些一直在你周围、一直等着你的来自上帝的魂灵们，你就永远不会孤独。祝福我所有的朋友们，祝福你们的身体、你们的灵魂以及你们的精神。愿你们内心平静，上帝祝福你们，与上帝同在！
_
_
©2019 International Pathwork Foundation
Translated by Mark. Edited by April.
微信订阅号 @Pathwork Lectures 心路讲座中文
